.github/workflows/deploy.yml

```diff
@@ -96,18 +95,31 @@ jobs:
         with:
           state: 'success'
           token: '${{ secrets.GITHUB_TOKEN }}'
-      - name: 'Fire Wildfly Operator examples build'
+      - name: Login Operator Quickstart quay.io registry
+        if: success()
+        run: docker login -u="${{ secrets.QUICKSTARTS_ROBOT_NAME }}" -p="${{ secrets.QUICKSTARTS_ROBOT_PASSWORD }}" quay.io
+      - name: Build PostgreSQL example and push to quay.io
         if: success()
         run: |
-          TARGET_REPO="https://api.github.com/repos/${WILDFLY_OPERATOR_REPO_USER}/wildfly-operator"
-          echo "Invoking building examples images workflow on ${TARGET_REPO}"
-          curl \
-          -X POST \
-          -H "Authorization: token ${WILDFLY_OPERATOR_GITHUB_PAT}" \
-          -H "Accept: application/vnd.github.ant-man-preview+json" \
-          -H "Content-Type: application/json" \
-          ${TARGET_REPO}/dispatches \
-          -d '{ "event_type" : "build_operator_examples", "client_payload" : { "source" : "wildfly-s2i" }}'
+          s2i build https://github.com/wildfly/wildfly-s2i \
+          -e GALLEON_PROVISION_LAYERS=jaxrs-server,postgresql-datasource,observability \
+          -r master \
+          --context-dir test/test-app-postgres \
+          --loglevel=5 \
+          quay.io/wildfly/wildfly-centos7:latest \
+          wildfly-s2i-builder-image
```

Comment:
Shouldn't the name of the built image reflect the example name? Something like test-app-postgres-img? Same for the next example.

Reply:
I didn't see the need. It's an image built and pushed to the ephemeral container kicked off by the GitHub Actions. At first glance, I thought we could save space if we reuse the same name for the builder image on each example, it also facilitates the copy&paste ... well, those are not good reasons :) so, if you prefer and/or think there are advantages, I can rename them

Reply:
These are good reasons, that is fine with me!